src/helpers/retry.ts

```diff
@@ -0,0 +1,25 @@
+import retry from 'async-retry'
+
+const errorCodesNoRetry = [400, 403, 413]
+
+export const retryRequest = async (requestBuilder: () => Promise<any>, retryOpts: retry.Options): Promise<void> => {
+  // Request function, passed to async-retry
+  const doRequest = async (bail: (e: Error) => void) => {
+    try {
+      await requestBuilder()
```

Comment:
I think it was not introduced by your PR, however the naming `request builder` is used to designate functions to build "functions that will perform queries", which does not seem to be the case here as it is directly a function performing a request.
I'd suggest to rename it to `requestPerformer`